quantum-adapters/kiwoom-adapter/src/kiwoom_api/functions/new_stock_rights.py

```
@@ -0,0 +1,199 @@
+"""키움 신주인수권전체시세요청(ka10011) 비즈니스 로직"""
+
+import logging
+import httpx
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)